### DISCUSSION TITLE
如何给Dropdown组件添加遮罩，然后对其点击，dropdown可以关闭？

### DISCUSSION THREAD

**Discussion by liaodalin19903** [ant-design/ant-design#45034] [Q&A] [answered]
我在使用[dropdown](https://ant-design.antgroup.com/components/dropdown-cn)的时候，在它open的时候想要对其添加一个背景遮罩，然后点击遮罩它就关闭，请问如何添加呢？

![image](https://github.com/ant-design/ant-design/assets/12727582/a21dc8d2-4612-44c6-8b48-f8cd3569c66a)
我寻找了[dropdown组件的API](https://ant-design.antgroup.com/components/dropdown-cn)，没有发现有对应的API能做到。

**Comment by Wxh16144** [comment]
默认不就是点击外面关闭吗？

**Comment by liaodalin19903** [comment]
给做了open状态，需要触发事件去改变状态。所以在这个场景下。



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2023年09月24日 00:25 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [ant-design/ant-design] 如何给Dropdown组件添加遮罩，然后对其点击，dropdown可以关闭？ (Discussion #45034) |

默认不就是点击外面关闭吗？

**Comment by liaodalin19903** [comment, marked as answer]
我找到了答案，可以在这里面进行更新。
```
 <Dropdown 
onOpenChange={()=>{
  // 更新逻辑（关闭dropdown）
}}
```
